net-voip/telepathy-haze/telepathy-haze-0.8.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6..9} pypy3 )
+
+inherit python-any-r1
+
+DESCRIPTION="Telepathy connection manager providing libpurple supported protocols"
+HOMEPAGE="https://telepathy.freedesktop.org https://developer.pidgin.im/wiki/TelepathyHaze"
+SRC_URI="https://telepathy.freedesktop.org/releases/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm arm64 ~ia64 ~ppc ~ppc64 ~sparc x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=net-im/pidgin-2.7[dbus]
+	>=net-libs/telepathy-glib-0.15.1
+	>=dev-libs/glib-2.30:2
+	>=dev-libs/dbus-glib-0.73
```

Comment:
Please order alphabetically

Reply:
Also you can remove >= for: pidgin, telephathy-glib, glib and dbus-glib, because they don't exist anymore on repo (at these releases I mean). So it would be:

dev-libs/dbus-glib
dev-libs/glib
net-im/telephaty-glib
net-im/pidgin